packages/components/src/components/hds/application-state/header.hbs

```diff
@@ -3,18 +3,18 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 <div class="hds-application-state__header" ...attributes>
-  {{#if @icon}}
-    <div class="hds-application-state__icon">
-      <FlightIcon @name={{@icon}} @size="24" />
-    </div>
-  {{/if}}
-  <Hds::Text::Display class="hds-application-state__title" @tag="div" @size="400" @weight="semibold">
-    {{@title}}
-  </Hds::Text::Display>
   {{#if @errorCode}}
     <Hds::Text::Body class="hds-application-state__error-code" @tag="div" @size="100" @weight="medium">
       Error
       {{@errorCode}}
     </Hds::Text::Body>
   {{/if}}
+  {{#if @icon}}
+    <div class="hds-application-state__icon">
+      <FlightIcon @color="var(--token-color-foreground-faint)" @name={{@icon}} @size="24" />
```

Comment:
```suggestion
      <FlightIcon @color="var(--token-color-foreground-strong)" @name={{@icon}} @size="24" />
```

Reply:
This makes me think: should we (could we) add a `@color` argument to the `Icon` too, similar to what we already do for the `Text` component? https://github.com/hashicorp/design-system/blob/main/packages/components/src/components/hds/text/types.ts#L8-L29 (and potentially share these values between the two components)?